etabotsite/etabotapp/misc_utils/django_indexhtml_mod.py

```diff
@@ -0,0 +1,45 @@
+"""
+purpose: modifies index.html for django template
+		 by adding static files directives
+
+Author: Alex Radnaev
+
+date created: 2018-05-27
+
+"""
+import re
+import logging
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+def html_to_django_template(
+	src_html,
```

Comment:
Use space instead of tab according to PEP8 code styling

Reply:
fair enough, but Richard would disagree! :)